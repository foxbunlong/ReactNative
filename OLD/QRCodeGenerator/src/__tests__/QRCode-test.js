import React from 'react'
import QRCode from '../index'
import renderer from 'react-test-renderer'

describe('QRCode', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <QRCode />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('renders with logo correctly', () => {
    const tree = renderer.create(
      <QRCode
        logo={{ uri: 'fakeUri' }}
      />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
