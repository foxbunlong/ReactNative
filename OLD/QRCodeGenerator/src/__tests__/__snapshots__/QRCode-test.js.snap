// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QRCode renders correctly 1`] = `
<div>
  <div>
    <span>
      Rect
    </span>
    <span>
      Path
    </span>
  </div>
</div>
`;

exports[`QRCode renders with logo correctly 1`] = `
<div>
  <div>
    <span>
      Rect
    </span>
    <span>
      Path
    </span>
  </div>
  <div>
    <div>
      <span>
        Source: 
        fakeUri
      </span>
      <span>
        ResizeMode: 
        contain
      </span>
    </div>
  </div>
</div>
`;
